### PR TITLE
Remove unused method

### PR DESCRIPTION
**Pull request by josegonzalez**
This used to be used for generating `Security.salt` and `Security.cipherSeed`, but the method is unused in 3.x

### CODE REVIEW

File: src/Utility/Security.php [hunk 1]
```diff
@@ -48,16 +48,6 @@ class Security
     protected static $_instance;
 
     /**
-     * Generate authorization hash.
-     *
-     * @return string Hash
-     */
-    public static function generateAuthKey()
```
Comment (markstory) [inline src/Utility/Security.php]: This is a breaking change at this point. We forgot to deprecate this during the 3.0 development :cry:

Comment (markstory) [inline src/Utility/Security.php]: Given the issues in #6139 we could nerf this method and throw exceptions. While this is also a breaking change, it is an easier to debug one than simply removing the method.

Comment (josegonzalez) [inline src/Utility/Security.php]: So do you want me to raise an exception? You can just update the pr I guess...

Comment (markstory) [inline src/Utility/Security.php]: I think an exception would be best here.

**Pull request merged**